functions/go/export-terraform/terraformgenerator/templates/log-export.tf

```
@@ -45,4 +45,14 @@ module "{{ .GetResourceName }}-destination" {
     is_locked             = {{ .GetBool "spec" "retentionPolicy" "isLocked" }}
   }{{end}}
 }
+{{end}}{{with $logsink.References.LoggingLogBucket }}
+module "{{ .GetResourceName }}-destination" {
+  source  = "terraform-google-modules/log-export/google//modules/logbucket"
+  version = "~> 7.3.0"
```

Will only land in the next release
```suggestion
  version = "~> 7.4.0"
```

---

ack